frontend/simple/js/state.js

```diff
@@ -118,7 +171,7 @@ const actions = {
 
     const type = entry.type
     entry = Events[type].fromObject(entry, hash)
-    if (entry instanceof Events.HashableContract) {
+    if (entry instanceof Events.HashableContract && !state.contracts[contractId]) {
```

Comment:
If you perform this action on a contract that you already have subscribed to you'll get duplicate events. This is relevant when streaming a contracts events to synchronize.